archesproject/arches-for-science-prj
# Show unique top concepts for AAT concepts in search

All AAT Entry concepts are nested under a single top concept called 'Top Concept'. This causes all concepts listed in search to have the same context ('Top Concept'). Removing the 'Top Concept' makes all of the concepts below it top concepts providing some context in the search dropdown.

Resolves archesproject/arches#10387

To test this PR:

- Confirm that you see duplicate concepts, `painting (image-making) (Top Concept)`, in search by searching for 'painting' and confirm that you have a 'Top Concept' concept under AAT Entries in the RDM.
- Run migrations and reindex your database.
- Confirm that Top Concept is removed from under AAT Entries and that searching for 'painting' now shows a unique top concept for each `painting (image-making)` concept.
